### DISCUSSION TITLE
Circular progress and progress loading prop

### DISCUSSION THREAD

**Discussion by ashwinKumar0505** [chakra-ui/chakra-ui#2133] [General] [answered]
Was the isIndeterminate prop removed from the circular progress and progress components in the next-chakra-ui?
Is there any other way to make the component loading?
![Screenshot 2020-10-01 at 12 04 10 PM](https://user-images.githubusercontent.com/53852911/94819220-a17f4500-041c-11eb-9952-392296e33481.png)

**Comment by waptik** [comment, marked as answer]
upload the screenshot again.
The `isIndeterminate` props is still there as i use it and it works:
Here's my code 
```js
<CircularProgress aria-label="Loading" isIndeterminate color="blue.500"  size="220px" />
```

**Reply by ashwinKumar0505** [reply]
I had uploaded it again

**Reply by ashwinKumar0505** [reply]
https://codesandbox.io/s/xenodochial-meadow-jbcjo?file=/src/App.js

This is the codesandbox link

**Reply by waptik** [reply]
I tried the cbs you sent but i don't know why it doesn't work there.
See this one https://codesandbox.io/s/basic-chakra-template-lx861?file=/src/App.tsx

**Reply by ljosberinn** [reply]
it was gone for 1 or 2 rcs but has been re-added since :)
